.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: java
+jdk:
+    - oraclejdk7
```

Comment:
Looks like Travis-CI only supports openjdk6 for java 6 (java 6 is EOL)
This commit log is going to be a mess